bridge/src/mock.rs

```diff
@@ -363,12 +454,14 @@ impl sygma_bridge::Config for Runtime {
 	type ExtractDestData = DestinationDataParser;
 	type PalletId = SygmaBridgePalletId;
 	type PalletIndex = BridgePalletIndex;
+	type DecimalConverter = SygmaDecimalConverter;
+	type AssetDecimalPairs = AssetDecimalPairs;
```

Comment:
Then our config will be like this:

```suggestion
	type DecimalConverter = SygmaDecimalConverter<SygmaDecimalConverter>;
```